extensions.json

```
@@ -720,6 +720,10 @@
       "id": "JeffersonQin.latex-snippets-jeff",
       "repository": "https://github.com/JeffersonQin/VSCode-LaTeX-Snippets"
     },
+    {
+      "id": "jeroen-meijer.pubspec-assist",
+      "repository": "https://github.com/jeroen-meijer/pubspec-assist"
```

It's probably safer to pin the `v2.1.0` tag & version, instead of publishing from the default branch every night:

```suggestion
      "repository": "https://github.com/jeroen-meijer/pubspec-assist",
      "version": "2.1.0",
      "checkout": "v2.1.0"
```